code/modules/clothing/shoes/cowboy.dm

```diff
@@ -45,19 +45,17 @@
 	. = ..()
 	if(!(user.mobility_flags & MOBILITY_USE) || user.stat != CONSCIOUS || HAS_TRAIT(user, TRAIT_HANDS_BLOCKED) || !Adjacent(user) || !user.Adjacent(target) || target.stat == DEAD)
 		return
-	if(occupants.len >= max_occupants)
+	if(contents.len >= max_occupants)
 		to_chat(user, span_warning("[src] are full!"))
 		return
 	if(istype(target, /mob/living/simple_animal/hostile/retaliate/snake) || istype(target, /mob/living/simple_animal/hostile/headcrab) || istype(target, /mob/living/carbon/alien/larva))
-		occupants += target
 		target.forceMove(src)
 		to_chat(user, span_notice("[target] slithers into [src]."))
 
 /obj/item/clothing/shoes/cowboy/container_resist_act(mob/living/user)
 	if(!do_after(user, 10, target = user))
 		return
-	user.forceMove(user.drop_location())
-	occupants -= user
+	user.forceMove(src.drop_location())
```

Comment:
src?

Reply:
you should be dropped at the boot's drop location, not the user (who is inside the boot)

Reply:
basically it's the same thing, but imo, more correct.

Reply:
why not just drop_location then?

Reply:
I always add src on the proc's to be explicit tbh

Reply:
just make a change susggestion i'll add it